_includes/tutorials/transforming/kstreams/markup/answer/short-answer.adoc

```diff
@@ -0,0 +1,6 @@
+Use the `map()` method to take each input record and create a new stream with transformed records in it. The example below uses a custom function `convertRawMovie()` to perform the transformation logic.
```

Comment:
@awalther28 "perform the transformation logic" may sound a bit vague to users.  I defer to @bbejeck and you for more accurate wording, but maybe rephrase to something along the lines of "convert the object type"

Reply:
I reworded this to --> "Use the `map()` method to take each input record and create a new stream with transformed records in it. The records are transformed via a custom function, in this case `convertRawMovie()`." 

But maybe that's still a little too vague 🤷‍♀️